### PR TITLE
Documentation: Refer back between name and initial-cluster options

### PR DESCRIPTION
**Pull request by ecnahc515**
Re-opened #2997 under my fork.

### CODE REVIEW

File: Documentation/configuration.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 ##### -name
 + Human-readable name for this member.
 + default: "default"
++ This value is referenced as one of this node's own entries listed in the `-initial-cluster` flag (Ex: `default=http://localhost:2380` or `default=http://localhost:2380,default=http://localhost:7001`). This needs to match the key used in the flag if you're using [static boostrapping](clustering.md#static).
```
Comment (yichengq) [inline Documentation/configuration.md]: remove `one of`? it can appear multiple times.

Comment (ecnahc515) [inline Documentation/configuration.md]: Fixed

**Comment by ecnahc515** [comment]
@yichengq Updated.

**Comment by yichengq** [comment]
lgtm

**Pull request merged**